packages/sw-routing/src/lib/route.js

```diff
@@ -72,20 +70,27 @@ class Route {
    *        and `event`, which is a [FetchEvent](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent).
    *        `match` should return a truthy value when the route applies, and
    *        that value is passed on to the handle function.
-   * @param {Object} input.handler An Object with a `handle` method. That
-   *        function is passed an object with the same `url` and `event`
-   *        properties as `match` received, along with an additional property,
-   *        `params`, set to the truthy value that `match` returned.
+   * @param {Object|function} input.handler An Object with a `handle` method, or
```

Comment:
Nit: Could you swap this around to function|Object and the comment.

Basically I expect most developers will never care about this because they'll use a default caching strategy, but if they do use this I imagine function will be the preferred way.